### DISCUSSION TITLE
ngModel for datepicker

### DISCUSSION THREAD

**Discussion by Roghayeh-Dehghani** [SaeedDev94/ng-persian-datepicker#62] [Q&A]
Hi,

i want to use datepicker in  ngFor and i need ngModel to bind  it . 
i would be appreciated if you could  help me.

         <ng-persian-datepicker  [uiTheme]="uiTheme" [timeEnable]="timeEnable" [timeMeridian]="timeMeridian" 
               [uiTodayBtnEnable]="uiTodayBtnEnable" [dateFormat]="dateFormat" >
                            <input type="text" class="EInputText"  [(ngModel)]="personnel.deliveryDate"  !important" />
           </ng-persian-datepicker>

**Comment by SaeedDev94** [comment]
Hi, currently `ng-persian-datepicker` doesn't support `[(ngModel)]`,
You should use `FormArray` for loop
